### ISSUE TITLE
More metadata about providers

### ISSUE THREAD

**Issue by pierreozoux** [nextcloud/nextcloud.com#419]
If we want to rank providers as discussed in https://github.com/nextcloud/android/issues/53, we need to gather more metadata about said providers.

Here is a proposed list:
- [x] language
- [x] description
- [x] logo
- [ ] beginning operation date
- [ ] timezone support - opening hours
- [ ] price
- [ ] free software only
- [ ] is a Chatons
- [ ] charts ( https://userdatamanifesto.org/ )
- [ ] uptime
- [ ] tls quality / security
- [ ] user feedback

(Listed as done/todo and in order of difficulty to implement)

This issue is actually similar to what https://chatons.org is solving, so we could find ways to collaborate.

It would be nice to gather a first round of feedback about those metadata, and then opening one issue for each on ways to implement them.

@Gomez @jancborchardt @jospoortvliet @mabkenar @owndrive

**Comment by MariusBluem** [comment]
We will start with some requirements for the providers page in the coming days: https://github.com/nextcloud/nextcloud.com/issues/395

your points seem to be very complicated to prove and to monitor ... in some cases 😞 

> price

I don't want to have pricing stuff on Nextcloud page, this is why pricing stuff will be not longer allowed in the provider-descriptions (see requirements mentioned above). Otherwise, it smells to me too much like advertising :)

> free software only

Free-software only is also complicated because no-one would say - that he does not fulfill this 😏😜 Hard and simply not possible to check 🙃

> uptime
> tls quality / security 

Let's don't start running uptime, advanced security etc. checks against providers - because otherwise the users are right in their point of view to rely on this information - and this is the point we would take over responsibility for the providers - **and this is definitely not what we want** 😜😜🙈🙈

**Comment by jospoortvliet** [comment]
Agreed with Marius - but the other points are useful. This really is complicated. There have been some heavy complaints about some providers which, at the same time, are getting a lot of positive feedback. An example is ownCube, where some ppl tell me "remove them from the provider page, they are bad" and then others love what they do and they provide an amazingly popular product with some cool new features (they now offer collabora, Spreed...).

We have a rich diversity of providers, I want to keep that. But also keep OUT rotten eggs. It is hard... ;-)
